src/components/Proposal/Create/CreateProposalPage.tsx

```diff
@@ -62,8 +62,13 @@ class CreateProposalPage extends React.Component<IProps, IStateProps> {
     history.push("/dao/" + daoAvatarAddress + "/plugin/" + pluginId);
   }
 
+  private preventDefault = (e: any) => {
```

Comment:
I don't see this being used

Reply:
Yes I noticed this later and removed it in the last push.